__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const data = require("../db/data/test-data/index.js")
+const request = require("supertest")
+const app = require("../app.js")
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed.js')
+beforeEach(() =>  seed(data))
+afterAll(() => db.end())
+
+describe('GET /api/topics', () => {
+    test('should return a 200 status code for a good request', () => {
+        return request(app).get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            
+            response.body.topics.forEach((item) => {
```

Comment:
If you're going to forEach through something within your test file you should check the length of the array you're getting back first as you may get a false positive.